src/test/HubApiCall_Disconnect.steps.ts

```diff
@@ -0,0 +1,135 @@
+import {defineFeature, loadFeature} from "jest-cucumber";
+import {HubApiCall} from "../routes/oath";
+const axios = require("axios");
+const Disconnect = loadFeature('features/Disconnect.feature');
+const querystring = require('query-string');
+axios.defaults.adapter = require('axios/lib/adapters/http')
+const correctCall = async () => {
+    try {
+        // this represents any api call that doesn't fail
+       const response = await axios.get('https://dog.ceo/api/breeds/list/all')
+        const data = response.data;
+        return data;
+    } catch (e) {
+        console.error('  > Unable to retrieve contact');
+        return JSON.parse(e.response.body);
+    }
+
+}
+
+const errorCall = async () => {
+    try {
+        var options = {
+            method: 'GET',
+            url: 'https://api.hubapi.com/crm/v3/objects/contacts',
+            qs: {limit: '10', archived: 'false'},
+            headers: {accept: 'application/json', authorization: 'expired_refresh_token'}
+        };
+
+        const response = await axios.get(querystring.stringify(options));
+        const data = response.data;
+        return data;
+
+    } catch (e) {
+        console.log('  > Unable to retrieve contact');
+        return JSON.parse(e.response.body);
+    }
+
+}
+
+defineFeature(Disconnect, test => {
+    //#todo UPDATE LINE BELOW TO RECIEVE access and refresh token from DB
+    let refresh_token: string, access_token: string;
+
+    test('Disconnecting Hubspot from integration', ({given, and, when, then}) => {
+
+        given('I have a SaaSquatch account', () => {
+
+        });
+
+        and('a Hubspot account', () => {
+
+        });
+
+        given('I have installed the integration in Hubspot', () => {
+            // this can be verified by checking if the user has a refresh token.
+        });
+
+        when('I disconnect the integration from my Hubspot account', () => {
+            // this implies that the refresh token has been invalidated
+            // unable to get selenium working as of yet but for testing
+            // purposes we can use an invalid refresh token (random string)
+            refresh_token = 'invalid token example';
+        });
+
+        and('the current access token expires', () => {
+            //
+            access_token = "nowexpiredaccess_token";
+        });
+
+        then('requests from the integration to protected endpoints in Hubspot fail', () => {
+            // the api call will happen in the final then
```

Comment:
Make a standard request here without the HubApiCall function to show it will fail?

Reply:
For some reason if I make any api call from here I get this error: 
![image](https://user-images.githubusercontent.com/42298613/123026800-03f8e680-d39a-11eb-9bc7-ed3560506d9b.png)

Reply:
Added some code as suggested